src/Tizen.Content.Download/Tizen.Content.Download/Request.cs

```diff
@@ -843,6 +844,213 @@ private void UnregisterProgressChangedEvent()
+            }
+        }
+
+        /// <summary>
+        /// Returns the 'enabled' state of the cache feature.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <remarks>
+        /// Get Enable/disable for _downloadId
+        /// </remarks>
+        /// <exception cref="ArgumentException">Thrown when it is failed due to an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when it is failed due to an invalid operation.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when a permission is denied.</exception>
+        /// <exception cref="NotSupportedException">Thrown when not supported.</exception>
+        public bool GetCacheEnableState()
+        {
+            bool enable;
+            int ret = Interop.Download.GetDownloadCache(_downloadId, out enable);
+            if (ret != (int)DownloadError.None)
+            {
+                DownloadErrorFactory.ThrowException(ret, "Failed to get cache enable state");
+            }
+            return enable;
+        }
+
+        /// <summary>
+        /// Clears the cache.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <exception cref="ArgumentException">Thrown when it is failed due to an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when it is failed due to an invalid operation.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when a permission is denied.</exception>
+        /// <exception cref="NotSupportedException">Thrown when not supported.</exception>
+        public void ResetCache()
+        {
+            int ret = Interop.Download.ResetDownloadCache();
+            if (ret != (int)DownloadError.None)
+            {
+                DownloadErrorFactory.ThrowException(ret, "Failed to reset cache");
+            }
+        }
+
+        /// <summary>
+        /// Sets the max size of the cache.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <param name="size">The max size of the cache (MB)</param>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <remarks>The default size is 1000 MB.</remarks>
+        /// <exception cref="ArgumentException">Thrown when it is failed due to an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when it is failed due to an invalid operation.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when a permission is denied.</exception>
+        /// <exception cref="NotSupportedException">Thrown when not supported.</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal void SetCacheMaxSize(uint size)
```

Comment:
Make type 'public' to be called from an user.
(This API is for internal use, only authorized users can use it.)

internal void SetCacheMaxSize(uint size) > public void SetCacheMaxSize(uint size)

Reply:
For set/get APIs, how about using C# Property? Please refer to 'public string DestinationPath {get; set;}'

Reply:
Changes methods to C# properties.